plan_delegator/test/test_plan_delegator.cpp

```diff
@@ -25,6 +25,10 @@
 
 namespace plan_delegator{
 
+    // These tests has been temporarily disabled to support Continuous Improvement (CI) processes.
+    // Related GitHub Issue: <https://github.com/usdot-fhwa-stol/carma-platform/issues/2335>
```

Comment:
Looks like we need to move this comment down now to the two tests that are commented out

Reply:
removed